Makefile

```diff
@@ -4,9 +4,9 @@ CXX=clang++-10
 CFLAGS=-Wall -fPIC
 CXXFLAGS=-std=c++17 -Wall -fPIC
 LDFLAGS=-lpthread -pthread
-NVCCFLAGS=--gpu-architecture=sm_60 -lineinfo --maxrregcount=63
+NVCCFLAGS=--std=c++17 --gpu-architecture=sm_60 -lineinfo --maxrregcount=63
```

Comment:
Not 100% sure if this change is necessary because it seemed to work fine for me before. However, I'm not exactly sure what happened but it would no longer compile unless I used this flag.